webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3586,7 +3586,8 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                 $mck_msg_sbmt.attr('disabled', false);
                 $applozic('.' + randomId + ' .mck-message-status').removeClass('mck-sent-icon').addClass('mck-pending-icon');
                 mckMessageLayout.addTooltip(randomId);
-                mckMessageLayout.clearMessageField(true);
+                var keyboard = (kommunicateCommons.isObject(messagePxy.metadata) && messagePxy.metadata.hasOwnProperty('keyboard')) ? messagePxy.metadata.keyboard : true ;
```

Comment:
one approach can be to add some specific data type on send button on click of rich message to notify when to not open the keyboard